aws/table_aws_backup_recovery_point.go

```diff
@@ -288,3 +297,62 @@ func getAwsBackupRecoveryPoint(ctx context.Context, d *plugin.QueryData, h *plug
 
 	return detail, nil
 }
+
+func getAwsBackupRecoveryPointTags(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	arn := recoveryPointArn(h.Item)
+
+	// Define the regex pattern for the recovery point ARN
+	pattern := `arn:aws:backup:[a-z0-9\-]+:[0-9]{12}:recovery-point:*`
```

Comment:
I think there should be a dot to denote "any character that follows the colon".
```suggestion
	pattern := `arn:aws:backup:[a-z0-9\-]+:[0-9]{12}:recovery-point:.*`
```